server/controllers/finance/patientInvoice.js

```diff
@@ -293,3 +297,99 @@ function search(req, res, next) {
   .catch(next)
   .done();
 }
+
+/** 
+ * @function invoiceStat
```

Comment:
Can this function be moved to a different controller? After brief discussion it seems like it would be good to keep these controllers for basic CRUD operations. This file is also getting very large and has a lot of different responsibilities. 

My dashboard controller can be found at `server/controllers/dashboard`. 

Following the naming scheme here I would suggest creating `stats.js` in this folder and creating an `invoiceStat` and `patientStat` method inside this.